cli/src/main.rs

```diff
@@ -92,3 +102,12 @@ as follows:"
     println!("    IROHA_GENESIS is the genesis block config");
     println!("Examples of these variables can be found in the default `configs/peer/config.json`.")
 }
+
+#[allow(clippy::print_stdout)]
+fn print_version() {
+    println!(
+        "iroha2 {} ({})",
```

Comment:
```suggestion
        "iroha {} ({})",
```

Version will already include 2, so maybe just "iroha"?

Reply:
Very good point